### PR TITLE
Pass project_name to base image pipelines

### PR DESCRIPTION
**Pull request by codificat**
Otherwise it takes the default value from the template, `lifecycle`...

Also reformatting a bit for (hopefully) better readability.

### CODE REVIEW

File: ansible/roles/jenkins/tasks/main.yml [hunk 1]
```diff
@@ -50,10 +50,22 @@
...
+    --param APP_NAMESPACE={{ project_name }}
+    | {{ oc }} apply --force=true -f -
+- name: Create application base image pipeline
+  # Using the same template but changing some parmeters to match the app
+  shell: >-
+    {{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml
+    --param BASE_IMAGE_TAG='nodejs:6'
+    --param APP_BC='nodejs-mongo-persistent'
+    --param APP_NAME='nodejs-mongo-persistent'
+    --param APP_PIPELINE='lifecycle-app-lifecycle'
```
Comment (jcpowermac) [inline ansible/roles/jenkins/tasks/main.yml]: @aweiteka is that right name of the pipeline?  https://github.com/RHsyseng/jenkins-on-openshift/blob/f841e6bb4743e4dde4b70d9bdf3a1159eea381a6/ansible/roles/jenkins/files/app-pipeline.yaml#L10

Comment (jcpowermac) [reply ansible/roles/jenkins/tasks/main.yml]: Oh wait this is the name in Jenkins.  But isn't it a slash vs a hyphen?

Comment (jcpowermac) [reply ansible/roles/jenkins/tasks/main.yml]: nevermind, display name vs project name.

**Comment by jcpowermac** [comment]
LGTM

File: ansible/roles/jenkins/tasks/main.yml [hunk 2]
```diff
@@ -50,10 +50,22 @@
...
+  # Most of the defaults in the template are already set to match Jenkins
+  shell: >-
+    {{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml
+    --param APP_NAMESPACE={{ project_name }}
+    | {{ oc }} apply --force=true -f -
+- name: Create application base image pipeline
+  # Using the same template but changing some parmeters to match the app
+  shell: >-
+    {{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml
+    --param BASE_IMAGE_TAG='nodejs:6'
```
Comment (aweiteka) [inline ansible/roles/jenkins/tasks/main.yml]: These should be params from `ansible/group_vars/all[.example]`

**Pull request merged**